frontend/packages/console-shared/src/utils/pod-ring-utils.ts

```diff
@@ -22,6 +23,27 @@ const applyPods = (podsData: PodRingData, dc: PodRCData) => {
   return podsData;
 };
 
+export const podRingLabel = (resourceObj: K8sResourceKind, isScalingAllowed: boolean) => {
+  const title =
+    resourceObj.status.availableReplicas || (isScalingAllowed ? 'Scaled to 0' : 'Autoscaled');
+
+  const subTitle =
+    resourceObj.spec.replicas !== resourceObj.status.availableReplicas
+      ? !resourceObj.spec.replicas
+        ? isScalingAllowed
+          ? ''
+          : 'to 0'
+        : `scaling to ${resourceObj.spec.replicas}`
+      : resourceObj.spec.replicas > 1 || resourceObj.spec.replicas === 0
+      ? 'pods'
+      : 'pod';
+
+  return {
```

Comment:
We shouldn't declare variables solely for the purpose of using ES6 object literal shorthand. Just cut out the extra variable declarations, and assign the `title` and `subTitle` properties directly in the return object.

Reply:
@vikram-raj ^^

Reply:
Done.